### PR TITLE
Fix backspace on iOS

### PR DESCRIPTION
**Pull request by swoolcock**
Holding backspace on software keyboard was triggering "delete word" at the iOS level but not in the framework. This means delete events were being slowed down by the operating system.  Also backspace was not being correctly handled by the hardware keyboard.

### CODE REVIEW

File: osu.Framework.iOS/IOSGameView.cs [hunk 1]
```diff
@@ -96,21 +96,29 @@ public override void SwapBuffers()
...
         {
             public event Action<NSRange, string> HandleShouldChangeCharacters;
             public event Action HandleShouldReturn;
             public event Action<UIKeyCommand> HandleKeyCommand;
 
-            public const int CURSOR_POSITION = 5;
+            /// <summary>
+            /// Placeholder text that the <see cref="HiddenTextField"/> will be populated with after every keystroke.
+            /// </summary>
+            private const string placeholder_text = "placeholder placeholder placeholder";
```
Comment (swoolcock) [inline osu.Framework.iOS/IOSGameView.cs]: Split to multiple words to ensure "fast delete" works properly.

File: osu.Framework.iOS/IOSGameView.cs [hunk 2]
```diff
@@ -96,21 +96,29 @@ public override void SwapBuffers()
...
         {
             public event Action<NSRange, string> HandleShouldChangeCharacters;
             public event Action HandleShouldReturn;
             public event Action<UIKeyCommand> HandleKeyCommand;
 
-            public const int CURSOR_POSITION = 5;
+            /// <summary>
+            /// Placeholder text that the <see cref="HiddenTextField"/> will be populated with after every keystroke.
+            /// </summary>
+            private const string placeholder_text = "aaaaaa";
```
Comment (peppy) [inline osu.Framework.iOS/IOSGameView.cs]: i thought this needed to be space separated? did you realise it was not required after all?

Comment (swoolcock) [reply osu.Framework.iOS/IOSGameView.cs]: Yeah not required.

Comment (peppy) [reply osu.Framework.iOS/IOSGameView.cs]: That makes this change easier to merge, cool.

**Pull request merged**